build.gradle

```diff
@@ -60,6 +60,13 @@ apply from: file('gradle/dependencies.gradle')
 
 repositories {
     jcenter()
+
+    // for r8 dependency
+    repositories {
```

Comment:
For what it's worth, you can also get stable releases from `google()`, but they're tied to AGP updates

Reply:
the latest one available from google() was 1.6.84 which does not include some classes that I need, mainly the JdkClassFileProvider.

I am happy to use published releases once they are available for my use-case.

Reply:
2.0.99 available on maven.google.com now